app/Http/Controllers/Admin/AdminController.php

```diff
@@ -17,4 +17,19 @@ public function index()
     {
```

Comment:
Please add git rule , not commit thosse '.DS_Store' file since it have no related to working source code

Reply:
Fixed